ApiBundle/Context/GroupContext.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Context;
+
+/**
+ * Class GroupContext
```

Comment:
wouldn't it be a good starting point for some code documentation as discussed some time ago ?